doc/code-docs/locales/en/LC_MESSAGES/checkpoint.po

```diff
@@ -20,7 +20,8 @@ msgstr ""
 "Generated-By: Babel 2.12.1\n"
 
 #: ../../source/checkpoint.rst:2
-msgid "模型保存"
+#, fuzzy
```

Comment:
If there are changes to the original Chinese RST files, the line `fuzzy` translation need to be deleted to make the English document rendered normally.

References: https://github.com/sphinx-doc/sphinx/issues/9618